docs/how-to-guides/query-api.md

```diff
@@ -68,6 +68,10 @@ Here's an example of how to use these clients:
         """
 
         async with AsyncLogfireQueryClient(read_token='<your_read_token>') as client:
+            # Get read token info
```

Comment:
I'm not sure we need to document this at all, but if we do I think it should be less prominent, not in this usage example. This is not something I expect to be used much, I don't even know why @samuelcolvin wants it.

Reply:
I imagine it might be useful for generic things like your export script. But I expect users to know where their read tokens come from, so I can't imagine what users would do with this.